modules/cms/formwidgets/Components.php

```diff
@@ -57,9 +58,16 @@ protected function listComponents()
                 }
             }
             catch (Exception $ex) {
-                $componentObj = new UnknownComponent(null, $properties, $ex->getMessage());
-                $componentObj->alias = $alias;
-                $componentObj->pluginIcon = 'icon-bug';
+                if ($name[0] === '@') {
```

Comment:
Can you use starts_with() instead of all these different ways of looking at the start of the string?